instrumentation/othttp/handler.go

```diff
@@ -120,6 +120,7 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.handler.ServeHTTP(rww, r.WithContext(ctx))
 
 	setAfterServeAttributes(span, bw.read, rww.written, rww.statusCode, bw.err, rww.err)
+	span.SetStatus(standard.SpanStatusFromHTTPStatusCode(int(rww.statusCode)))
```

Comment:
Should this be moved into the "post serve" function `setAfterServeAttributes`?

`rww.statusCode` is already an `int` no need for the conversion.

Reply:
> Should this be moved into the "post serve" function setAfterServeAttributes?

I thought about that as well, but it explicitly says it's setting attributes. v0v  I suppose the name could change or we could ignore the slight mismatch, doesn't bother me much.